### PR TITLE
Use redux for local tracks instead of conference.js

### PR DESCRIPTION
**Pull request by paweldomas**
This PR tries to get rid of `localAudio` and `localVideo` fields in conference.js

### CODE REVIEW

**Comment by saghul** [comment]
❤️❤️❤️

File: conference.js [hunk 1]
```diff
@@ -1356,12 +1344,16 @@ export default {
...
 
-            logger.error(
-                    'Ignored not an audio nor a video track: ', track);
+            logger.error('Ignored not an audio nor a video track: ', track);
 
             return Promise.resolve();
 
         });
+
+        return Promise.all(promises).then(() => {
```
Comment (jallamsetty1) [inline conference.js:1353]: Can we return `Promise.allSettled` instead ?

**Review by jallamsetty1 — APPROVED** [review]

**Comment by paweldomas** [comment]
the tests are failing and this time jvb failed to build, but previously one of the screen sharing tests has failed. I'll try again later...

**Comment by jallamsetty1** [comment]
> the tests are failing and this time jvb failed to build, but previously one of the screen sharing tests has failed. I'll try again later...

You can ignore the DesktopSharingTest.testAudioOnlyAndNonDominantScreenShare test, it is a known issue that we know of.

**Comment by paweldomas** [comment]
jenkins test this please

**Comment by paweldomas** [comment]
> You can ignore the DesktopSharingTest.testAudioOnlyAndNonDominantScreenShare test, it is a known issue and we are working on a fix.

oh okay

**Pull request merged**